jitsi-utils/src/main/java/org/jitsi/utils/PasswordUtil.java

```diff
@@ -71,7 +71,7 @@ public static String replacePasswords(String string, String[] passwordArgs)
     {
         for (String passArg : passwordArgs)
         {
-            if (StringUtils.isNullOrEmpty(passArg))
+            if (passArg == null || passArg.trim().isEmpty())
```

Comment:
I think we'd started to move _towards_ apache commons for these (and it doesn't look like it got removed), is there a reason to swap to the manual check here?

Reply:
jitsi-utils doesn't have a dependency on apache commons and I didn't want to pull it in for this _one_ usage.